pkg/system/phase2_creating.go

```diff
@@ -569,6 +569,27 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 				}}
 				util.MergeVolumeMountList(&c.VolumeMounts, &secretVolumeMounts)
 			}
+		case "noobaa-log-processor":
+			if c.Image != r.NooBaa.Status.ActualImage {
+				coreImageChanged = true
+				c.Image = r.NooBaa.Status.ActualImage
+			}
+			// adding the missing Env variable from default container
+			util.MergeEnvArrays(&c.Env, &r.DefaultCoreApp.Env)
+			r.setDesiredCoreEnv(c)
+
+			if r.NooBaa.Spec.LogResources != nil {
+				c.Resources = *r.NooBaa.Spec.LogResources
```

Comment:
`r.NooBaa.Spec.LogResources` is optional and might be `nil`. in that case use the defaults